src/ocamlorg_frontend/pages/about.eml

```diff
@@ -83,10 +83,19 @@ Layout.render
             Programming languages matter. They affect the reliability, security, and efficiency of the code you write,
             as well as how easy it is to read, refactor, and extend. The languages you know can also change how you
             think, influencing the way you design software even when you’re not using them.
+          </p>
+           <p>
+            <i>
+              The OCaml language is known for many things: it’s secure, it’s been around for a long time, and it’s
+              inextricably linked to camels.
+            </i>
           </p>
           <p>
             OCaml mixes power and pragmatism in a way that makes it ideal for building complex software systems. What
-            makes OCaml special is that it occupies a sweet spot in programming language design. It provides a
+            makes OCaml special is that it occupies a sweet spot in programming language design. Computer programming 
+            has become more complex over the years, and OCaml has developed tools to express that
+            complexity in simple yet powerful ways. As a result, OCaml is uniquely poised to adapt to modern challenges
```

Comment:
> OCaml has developed tools to express that complexity in simple yet powerful ways.

I'm not sure what this means: what tools are you referring to?

Reply:
I was referring to the many user centred tools that OCaml has, that address developers' needs rather than academics.